package.json

```diff
@@ -26,61 +28,61 @@
     "react-native-elements": "1.0.0-beta7",
     "react-native-fabric": "https://github.com/corymsmith/react-native-fabric#523a4edab3b2bf55ea9eeea2cf0dde82c5c29dd4",
     "react-native-fs": "^2.11.17",
-    "react-native-gesture-handler": "^1.2.1",
+    "react-native-gesture-handler": "^1.3.0",
     "react-native-hyperlink": "0.0.14",
     "react-native-image-pan-zoom": "^2.1.11",
     "react-native-image-picker": "^0.27.1",
     "react-native-modal": "^6.5.0",
     "react-native-push-notification": "^3.1.1",
     "react-native-qrcode": "^0.2.7",
+    "react-native-reanimated": "^1.0.1",
```

Comment:
I don't see this used anywhere, you should remove it and unlink it

Reply:
It's installed because [this component](https://github.com/jemise111/react-native-swipe-list-view) relies on it.

Reply:
Ok but I don't see `react-native-swipe-list-view` used anywhere either

Reply:
Wait that link is wrong, it’s [this library](https://github.com/react-native-community/react-native-tab-view) that depends on it (and that one is definitely used haha).